app/jobs/bulkrax/create_relationships_job.rb

```diff
@@ -87,8 +87,8 @@ def collection_parent_work_child
         ObjectFactory.new(
           attributes: attrs,
           source_identifier_value: nil, # sending the :id in the attrs means the factory doesn't need a :source_identifier_value
-          work_identifier: parent_entry.parser.work_identifier,
-          related_parents_parsed_mapping: parent_entry.parser.related_parents_parsed_mapping,
+          work_identifier: parent_entry&.parser&.work_identifier,
```

Comment:
when adding a work to an existing work or collection, the parent_entry doesn't exist. these values being nil didn't cause any errors for me though. 🙌🏾